sway/desktop/xdg_shell.c

```diff
@@ -117,6 +126,11 @@ static void set_fullscreen(struct sway_view *view, bool fullscreen) {
 	wlr_xdg_toplevel_set_fullscreen(surface, fullscreen);
 }
 
+static bool wants_floating(struct sway_view *view) {
+	// TODO
```

Comment:
This can return true if the view isn't resizable, ie. if the min size is equal to the max size.

Reply:
Can you please clarify this? The `wants_floating` function is called when a view is mapped so that the implementation can decide whether it's floating by default (eg. for dialogs).

Reply:
Yes. I'm just suggesting a way to implement this for xdg-shell.